package.json

```diff
@@ -60,30 +60,30 @@
     "write-file-atomic": "^4.0.0"
   },
   "devDependencies": {
-    "@npm/types": "^1.0.1",
+    "@npm/types": "^2.0.0",
     "@types/cross-spawn": "^6.0.2",
     "@types/eslint": "^8.0.0",
     "@types/fs-extra": "^11.0.0",
     "@types/inquirer": "^8.0.0",
     "@types/json5": "2.2.0",
     "@types/mocha": "^10.0.0",
     "@types/ncp": "^2.0.4",
-    "@types/node": "20.12.7",
+    "@types/node": "^22.7.5",
     "@types/rimraf": "^3.0.0",
     "@types/sinon": "^17.0.0",
     "@types/tmp": "^0.2.0",
     "@types/write-file-atomic": "^4.0.0",
-    "c8": "^9.0.0",
+    "c8": "^10.1.2",
     "cross-spawn": "^7.0.3",
     "fs-extra": "^11.0.0",
     "inline-fixtures": "^1.1.0",
     "js-green-licenses": "^4.0.0",
     "mocha": "^10.0.0",
     "sinon": "^17.0.0",
     "tmp": "0.2.3",
-    "typescript": "^5.4.3"
+    "typescript": "^5.6.3"
   },
   "peerDependencies": {
-    "typescript": ">=3"
+    "typescript": ">=5.6.3"
```

Comment:
Is there more information on my 5.6.3 is needed, some tools are incompatible with the latest and the project I am supporting (Puppeteer) can't update to the latest version yet. As the PeerDependecy is not met. 
![image](https://github.com/user-attachments/assets/fb34a33c-7dbe-47c0-837b-ba35476c432b)
I think updating to `>=5` would have been sufficient.

Reply:
That's fair - opened a PR to fix:
- https://github.com/google/gts/pull/909